common/lib/xmodule/xmodule/templates/problem/jsinput_response.yaml

```diff
@@ -161,7 +161,7 @@ data: |
       }
       </script>
         <p>What is the range of passengers that can fit on a 747 aircraft? Set the low and high bounds. (416 and 660 persons)</p>
-        <p class="sr" aria-hidden="true">
+        <p class="is-hidden" aria-hidden="true">
```

Comment:
I believe it's stuff that the author's script relies on, but isn't meant to be visible. Kinda silly... I guess I'll see if I can move this to JS variables/objects, etc.